lambdas/shared/tests/test_utils.py

```diff
@@ -13,13 +13,20 @@
 
 from t4_lambda_shared.utils import (
     IncompleteResultException,
+    get_available_memory,
     get_default_origins,
     make_json_response,
     query_manifest_content,
     separated_env_to_iter,
 )
 
 
+def test_get_available_memory():
+    mem = get_available_memory()
+    assert isinstance(mem, int), "Expected an int"
+    assert mem > 1, "Expected some memory"
+
+
```

Comment:
Doesn't seem that useful - should be safe to assume that `psutil` works as advertised. I'd just remove it.

Reply:
I see your point but then that function will lose test coverage and it's used in multiple lambda functions so I'd prefer to leave it as is. It's also useful if e.g. we want to switch to getting memory from an AWS env variable.